src/platform-implementation-js/driver-common/sidebar/constants.ts

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames';
+import { legacyIdMap } from '../../lib/idMap';
+
+const cssGlobal = 'inboxsdk__app_sidebar_waiting_platform';
+
+const enum SidebarClassName {
+  /** v1.1.0 CSS module selector, */
+  v1_1_0 = 'inboxsdk__a0oshmeH6N_2vLtc3DNT',
+  /** v1.1.1 CSS module selector, */
+  v1_1_1 = 'inboxsdk__BV_FA4qVT4qYxs3LbEU4',
+}
+
+export const sidebarWaitingPlatformSelector = `.${cssGlobal},
+  ${[SidebarClassName.v1_1_0, SidebarClassName.v1_1_1]
+    .map((x) => '.' + x)
+    .join(', ')},
+  .${
+    // Pre CSS module selector <1.1.0
+    legacyIdMap('app_sidebar_waiting_platform')
+  }` as const;
+
+export const sidebarWaitingPlatformClassName = cx(
+  cssGlobal,
+  SidebarClassName.v1_1_0,
+  SidebarClassName.v1_1_1,
+  legacyIdMap('app_sidebar_waiting_platform')
+);
```

Comment:
We need to add the current idMap value here so that our CSS applies to the element:
```suggestion
export const sidebarWaitingPlatformClassName = cx(
  cssGlobal,
  SidebarClassName.v1_1_0,
  SidebarClassName.v1_1_1,
  legacyIdMap('app_sidebar_waiting_platform'),
  idMap('app_sidebar_waiting_platform')
);
```

Reply:
oh, I see where you set the classname you're adding `idMap('app_sidebar_waiting_platform')` there. I think it's a little confusing it's not here instead but don't feel strongly.

Reply:
I have it separately to highlight that it's not related to this compatibility classname/selector. I could add a comment to that effect.

Reply:
a comment would be good

Reply:
[64ec92d](https://github.com/InboxSDK/InboxSDK/pull/895/commits/64ec92d4fb9da777d71cf908a8674f103d660c27)